app/controllers/WorkingGroups.java

```diff
@@ -830,21 +830,31 @@ public static Result publishProposal(
                     .getString("appcivist.services.etherpad.default.apiKey");
             EtherpadWrapper wrapper = new EtherpadWrapper(etherpadServerUrl, etherpadApiKey);
 
-            //Let's save the revision with no number, so etherpad can generate one by itself
-            wrapper.getEtherpadClient().saveRevision(proposal.getExtendedTextPad().getPadId());
-            Map revisions = wrapper.getEtherpadClient().listSavedRevisions(proposal.getExtendedTextPad().getPadId());
-
-            JSONArray savedRevisions = (JSONArray) revisions.get("savedRevisions");
             Integer newRevision = null;
-            if (savedRevisions != null && !savedRevisions.isEmpty()) {
-                newRevision = ((Long) savedRevisions.get(savedRevisions.size() - 1)).intValue();
-                proposal.addRevisionToContributionPublishHistory(newRevision);
+            try {
```

Comment:
Yes, there was an issue with this line  `wrapper.getEtherpadClient().saveRevision(proposal.getExtendedTextPad().getPadId())` when i tried to publish the proposal (change status from New to Published)